edb/schema/delta.py

```diff
@@ -1234,77 +1234,77 @@ def _propagate_if_expr_refs(
         schema: s_schema.Schema,
         context: CommandContext,
         action: str,
-    ) -> Tuple[s_schema.Schema, List[qlast.DDLCommand]]:
+        fixer: Optional[
+            Callable[[s_schema.Schema, ObjectCommand[so.Object], str,
+                      CommandContext, s_expr.Expression],
+                     s_expr.Expression]
+        ]=None,
+    ) -> s_schema.Schema:
         scls = self.scls
         expr_refs = s_expr.get_expr_referrers(schema, scls)
-        # Commands to be executed after the original change is
-        # complete
-        finalize_ast: List[qlast.DDLCommand] = []
 
         if expr_refs:
             ref_desc = []
-            for ref, fn in expr_refs.items():
+            for ref, fns in expr_refs.items():
+                really_apply = False
+
                 from . import functions as s_func
                 from . import indexes as s_indexes
                 from . import pointers as s_pointers
+                from . import constraints as s_cnstr
```

Comment:
Downstream imports aren't ideal, we should clean this up by doing some form of dispatch at some point.